client/dashboard/sites/backups/fields.tsx

```diff
@@ -0,0 +1,51 @@
+import { Icon } from '@wordpress/components';
+import { dateI18n } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import { cloud, globe, image } from '@wordpress/icons';
+import type { ActivityLogEntry } from '../../data/types';
+import type { Field } from '@wordpress/dataviews';
+
+function getIcon( item: ActivityLogEntry ) {
+	switch ( item.name ) {
+		case 'attachment__uploaded':
+			return image;
+		case 'plugin__updated':
+			return globe;
+		default:
+			return cloud;
+	}
+}
+
+export function getFields(): Field< ActivityLogEntry >[] {
+	return [
+		{
+			id: 'date',
+			label: __( 'Date' ),
```

Comment:
We'll need to pass getValue here for sorting. Something like getValue: ( { item } ) => item.published ?? ''